kdb+.html.markdown

```diff
@@ -0,0 +1,772 @@
+---
+language: kdb+
+contributors:
+    - ["Matt Doherty", "https://github.com/picodoc"]
+    - ["Jonny Press", "jonny.press@aquaq.co.uk"]
+filename: learnkdb.q
+---
+
+The q langauge and it's database component kdb+ were developed by Arthur
```

Comment:
Hi!  Fixed the grammar.  The terms kdb and q are often used interchangeably to describe the language and database, and kdb is the more distinctive term so I chose that rather than q here. For example a google search for q returns also sorts of stuff, where as kdb returns the results about the language.  That ok?